apps/mail/components/create/ai-chat.tsx

```
@@ -79,15 +80,16 @@ const ExampleQueries = ({ onQueryClick }: { onQueryClick: (query: string) => voi
-    <div className="mt-6 flex w-full max-w-xl relative flex-col items-center gap-2">
+    <div className="relative mt-6 flex w-full max-w-xl flex-col items-center gap-2">
       {/* First row */}
       <div className="no-scrollbar relative flex w-full justify-center overflow-x-auto">
         <div className="flex gap-4 px-4">
           {firstRowQueries.map((query) => (
             <button
               key={query}
               onClick={() => onQueryClick(query)}
-              className="shrink-0 whitespace-nowrap rounded-md bg-[#f0f0f0] p-1 px-2 text-sm text-[#555555] dark:bg-[#262626] dark:text-[#929292]">
+              className="shrink-0 whitespace-nowrap rounded-md bg-[#f0f0f0] p-1 px-2 text-sm text-[#555555] dark:bg-[#262626] dark:text-[#929292]"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)